cmd/hypper/upgrade.go

```diff
@@ -114,6 +122,12 @@ func newUpgradeCmd(cfg *action.Configuration, logger log.Logger) *cobra.Command
 						logger.Info(eyecandy.ESPrintf(settings.NoEmojis, "Release %q does not exist. Installing it now. :toolbox:\n", client.ReleaseName))
 					}
 					instClient := action.NewInstall(cfg)
+					// Set namespace for the install client
+					if settings.NamespaceFromFlag {
+						instClient.Namespace = settings.Namespace()
+					} else {
+						instClient.SetNamespace(ch, settings.Namespace())
+					}
```

Comment:
Aren't we missing setting the Config namespace with a call to `client.Config.SetNamespace(client.Namespace)`?
Analogous to: https://github.com/rancher-sandbox/hypper/blob/1a686aa3a18ebb10bbb2727baa28ea7e416ccc1a/cmd/hypper/install.go#L126

Reply:
I think the lack of this line is causing the bug I found.